_topics/en/countersign.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Countersign
+
+## Required.  At least one category to which this topic belongs.  See
+## schema for options
+categories:
```

Comment:
Could also be considered a *Privacy Enhancement* since this would allow a client to try to connect to a server without revealing its identity.

Reply:
Done